src/components/comments/comment.tsx

```diff
@@ -80,7 +80,7 @@ export function Comment({
 
     if (commentRef.current) {
       commentRef.current.style.backgroundColor = isHighlight
-        ? '#fdf5e4'
+        ? ''
         : 'transparent'
     }
```

Comment:
@yanncloud I think this whole block is needed any more now.

Reply:
of course :)